src/contains.js

```diff
@@ -0,0 +1,94 @@
+import {default as polygonContains} from "./polygonContains";
+import {default as distance} from "./distance";
+import {epsilon,radians} from "./math";
+
+var contains = {
+  Sphere: function() {
+    return true;
+  },
+  Point: function(feature, point) {
+    return distance(feature.coordinates, point) === 0;
+  },
+  MultiPoint: function(feature, point) {
+    var coordinates = feature.coordinates, i = -1, n = coordinates.length, f;
+    while (++i < n) {
+      f = { type: "Point", coordinates: coordinates[i] };
+      if (contains[f.type](f, point)) {
```

Comment:
To allow reuse without constructing a temporary object, you can break out per-type methods analogous to polygonContains for the other types. These methods take coordinates rather than objects.

```js
function pointContains(coordinates, point) {
  return distance(coordinates, point) === 0;
}
```